### DISCUSSION TITLE
How optimized is the build for ARM32?

### DISCUSSION THREAD

**Discussion by lucadici** [bluenviron/mediamtx#3578] [Questions] [answered]
### Question

**Objective**
- RTSP/H264 -> WebRTC proxy on Embedded Linux ARM imx6

**Configuration**
paths:
  camerah264:
    source: rtsp://127.0.0.1:8555/test

  all_others:

**CPU usage for webrtc client (browser)**
Around 50 %

**CPU Usage for client connected directly to gstreamer pipe (RTSP client)**
Around 20%

What are the compilation flags for ARM32?

**Comment by aler9** [comment, marked as answer]
Hello, if you want to tune the ARM compilation flags you can just recompile the server by following instructions in the README:
https://github.com/bluenviron/mediamtx?tab=readme-ov-file#cross-compile

in particular, ARM flags are explained in this page:
https://go.dev/wiki/GoArm

[a comment by an automated account is not shown]
